industry/foc/float/foc_ident.c

```diff
@@ -164,6 +164,8 @@ int foc_ident_res_run_f32(FAR struct foc_ident_f32_s *ident,
   int   ret  = FOC_ROUTINE_RUN_NOTDONE;
   float err  = 0.0f;
   float vref = 0.0f;
+  static float curr_sum  = 0.0f;
+  static float volt_sum  = 0.0f;
```

Comment:
why use static here?

Reply:
App calls foc_routine_ident_run_f32 every cycle to calculated resistance, to get the average value by accumulation, we should set curr_sum and volt_sum as static

Reply:
I think you're right, I will improve the design